### PR TITLE
Improved UX: Logos, Quick Settings, Notifications

### LINKED ISSUE #22: Setting: Notification on/off

### PR DESCRIPTION
**Pull request by SenF1**
Fix #22

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css2?family=Jockey+One&display=swap');
```
Comment (haok1402) [inline public/style.css]: Could we add fonts with `npm` and bundle within our build? Chrome Web Store requires additional reviews if we are importing codes from third-party libraries, even if just for fonts.

File: public/popup.js
Comment (haok1402) [inline public/popup.js:1]: Personally, I'd prefer any script-related files to be organized under `src`, what's your thoughts? @SenF1

File: src/modules/notification.ts
Comment (haok1402) [inline src/modules/notification.ts:1]: We're a step closer on organizations :tada:

File: public/popup.js
Comment (SenF1) [reply public/popup.js:1]: Sure, I dont't think it actually matters as everything will be in the root after build.

File: public/style.css (hunk 1 above)
Comment (SenF1) [reply public/style.css]: Hmm, I also tried to add Bootstrap with npm but that didn't work so I ended up using CDN. Can you take a look?

**Review by haok1402 — APPROVED** [review]

**Pull request merged**